addons/editor-theme3/theme3.js

```diff
@@ -392,4 +392,50 @@ export default async function ({ addon, console }) {
   addon.settings.addEventListener("change", updateColors);
   addon.self.addEventListener("disabled", updateColors);
   addon.self.addEventListener("reenabled", updateColors);
+
+  while (true) {
+    const colorModeSubmenu = await addon.tab.waitForElement(
+      "[class*=menu-bar_menu-bar-menu_] > ul > li:nth-child(2) ul",
+      {
+        markAsSeen: true,
+        reduxCondition: (state) => !state.scratchGui.mode.isPlayerOnly,
+      }
+    );
+
+    colorModeSubmenu.addEventListener(
+      "click",
+      (e) => {
+        if (addon.self.disabled) return;
+        if (!e.target.closest(".sa-colormode-submenu")) return;
+        if (e.target.closest(".sa-theme3-link")) {
+          window.open("https://scratch.mit.edu/scratch-addons-extension/settings#addon-editor-theme3");
+          e.stopPropagation();
+          return;
+        }
+        e.stopPropagation();
+      },
+      { capture: true }
+    );
+
+    const elementToClone = colorModeSubmenu.querySelector("[class*=settings-menu_selected_]").closest("li");
+
+    const managedBySa = elementToClone.cloneNode(true);
```

Comment:
Hmm, why clone?

Reply:
It was the fastest way to get a prototype done. Manually creating each element is also an option.